utils/crossref/transform/collection.js

```diff
@@ -19,13 +20,17 @@ const transformMetadata = (metadata, kind, timestamp) =>
 	});
 
 export default ({ globals, community }) => (collection) => {
-	const { title, metadata, attributions } = collection;
+	const { timestamp, dois, contentVersion } = globals;
+	const { crossrefDepositRecord, title, metadata, attributions } = collection;
 	return {
 		url: collectionUrl(community, collection),
 		...transformMetadata(metadata, collection.kind, globals.timestamp),
 		title: title,
-		timestamp: globals.timestamp,
+		timestamp: timestamp,
```

Comment:
In retrospect this design where you have to pass `timestamp` around like a hot potato is a little silly, sorry